modules/cms/ServiceProvider.php

```diff
@@ -55,6 +63,49 @@ public function boot()
         $this->bootRichEditorEvents();
     }
 
+    /**
+     * Register Twig templating engine.
+     */
+    protected function registerTwig()
+    {
+        App::singleton('cms.twig.loader', function () {
+            return new TwigLoader;
+        });
+
+        App::singleton('cms.twig.environment', function ($app) {
+            $loader = $app->make('cms.twig.loader');
+
+            $useCache = !Config::get('cms.twigNoCache');
+            $isDebugMode = Config::get('app.debug', false);
+            $strictVariables = Config::get('cms.enableTwigStrictVariables', false);
+            $strictVariables = is_null($strictVariables) ? $isDebugMode : $strictVariables;
```

Comment:
Was there a reason to switch from the short version (`$strictVariables = $strictVariables ?? $isDebugMode;`) to is_null verification?

Reply:
@SeriousKen ^

Reply:
@w20k to be honest I'm not sure why, but looking at that code `$strictVariables` is never `null`; it's either `true` or `false`, so neither line of code works as I imagine it was intended. Surely it should be `$strictVariables = $strictVariables ?: $isDebugMode`?

Reply:
@SeriousKen, could you change that part and also, if you don't mind add few Unit tests?